packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/dedicatedCloud-datacenter-backup.routes.js

```diff
@@ -1,7 +1,7 @@
 angular.module('App').config(
   /* @ngInject */ ($stateProvider) => {
-    $stateProvider.state('app.dedicatedClouds.datacenter.backup', {
-      url: '/backup',
+    $stateProvider.state('app.dedicatedClouds.datacenter.backup-old', {
+      url: '/backup-old',
```

Comment:
```suggestion
      url: '/backup-legacy',
```
sounds better

Reply:
This is just a temporary rename and these files will be removed at the end of dev for the new ranges. Further, we have another route for legacy, which we are working on

Reply:
I've unresolved so we know that this is to be done 🙂

Reply:
The old files have been removed at https://github.com/ovh/manager/pull/2385
Thanks :)